membership-attribute-service/app/actions/CommonActions.scala

```diff
@@ -32,7 +33,7 @@ class CommonActions(touchpointBackends: TouchpointBackends, bodyParser: BodyPars
 class BackendRequest[A](val touchpoint: TouchpointComponents, request: Request[A]) extends WrappedRequest[A](request)
 
 class AuthenticatedUserAndBackendRequest[A](
-    val user: Option[UserFromToken],
+    val user: Either[AuthenticationFailure, UserFromToken],
```

Comment:
I'm a bit confused as to why the authenticated request has an optional user here

Reply:
This is just preserving the type that's returned from the Identity lib so as not to break any other clients that are calling it.